package.json

```diff
@@ -40,16 +40,16 @@
     "**/qunit": "^2.14.1"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.11",
+    "@types/jest": "^26.0.23",
```

Comment:
If we can't update jest, shouldn't we also not update the types? Presumably they are adding types for features that we don't have?

Reply:
good catch